pkg/misc/slice/chunk.go

```diff
@@ -0,0 +1,16 @@
+package slice
```

Comment:
nit: missing the header on this file. `// Copyright 2024 The GUAC Authors.`..etc

Reply:
Fixed in 49f0d3ed3e0469c4d9595124b4ee10cd207414d1